paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheetViewModel.kt

```diff
@@ -161,6 +162,19 @@ internal class PaymentSheetViewModel @Inject internal constructor(
 
     private var deferredIntentConfirmationType: DeferredIntentConfirmationType? = null
 
+    private val googlePayButtonType: GooglePayButtonType =
```

Comment:
We could also move this to an internal property of the enum `enum class ButtonType(internal val googlePayButtonType: GooglePayButtonType)`.